settings.gradle

```diff
@@ -150,6 +152,7 @@ includeWithFlags ':examples:grpc-kotlin',                          'java11', 'ko
 includeWithFlags ':examples:grpc-scala',                           'java11', 'scala-grpc_2.13', 'scala_2.13'
 includeWithFlags ':examples:grpc-reactor',                         'java11', 'reactor-grpc'
 includeWithFlags ':examples:proxy-server',                         'java11'
+includeWithFlags ':examples:resilience4j-spring',                  'java17'
```

Comment:
I also want to add a pure Armeria example with Resilience4J. 😆 (Don't have to address in this PR)

Reply:
Will add in a follow-up PR 🙇